spec/helpers/cip_breadcrumb_helper_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe CipBreadcrumbHelper do
+  describe "core-induction-programme" do
```

Comment:
I was thinking more like 
```
RSpec.describe CipBreadcrumbHelper, type: :helper do
  describe "#year_breadcrumb" do
    ...
  end

  describe "#module_breadcrumb" do
    ...
  end

  describe "#lesson_breadcrumb" do
    ...
  end
end
```
And each describe can have a test for 'edit' action and a test for 'show' action.

Reply:
No problem, will add the others :)